libs/application/api-template-utils/src/lib/utils/assign.ts

```diff
@@ -0,0 +1,55 @@
+import jwt from 'jsonwebtoken'
+import { SendMailOptions } from 'nodemailer'
+import { dedent } from 'ts-dedent'
+
+import { Application } from '@island.is/application/core'
+
+export const createAssignToken = (application: Application, secret: string) => {
+  const token = jwt.sign(
+    {
+      applicationId: application.id,
+    },
+    secret,
+    { expiresIn: 24 * 60 * 60 },
+  )
+
+  return token
+}
+
+export const createAssignTemplate = (
+  application: Application,
+  email: string,
+  token: string,
+): SendMailOptions => {
+  const template = {
+    from: {
+      name: 'Fæðingarorlofssjóður',
+      address: 'baering@aranja.com',
+    },
+    replyTo: {
+      name: 'Fæðingarorlofssjóður',
+      address: 'baering@aranja.com',
```

Comment:
maybe add todo to be able to set this via parameters in the function. This functionality will be used by other applications than Fæðingarorlofssjóður

Reply:
Yeah sorry, this is an outdated file that I forgot to remove. Email generators are now a part of the templates.